src/render.ts

```diff
@@ -0,0 +1,24 @@
+import { GameState, CellValue } from "./gamestate";
+
+export function displayGameState(gameState: GameState) {
+  const renderedGameState = getRenderedGameState(gameState);
+  console.log(renderedGameState);
+}
+
+export function getRenderedGameState(gameState: GameState): string {
+  const spaceCharacter = " ";
+  const lineSeparatorCharacter = "\n";
```

Comment:
extract constants outside of the function (-> each time the function is called these variables are built again)